enricher/standard/src/test/java/io/fabric8/maven/enricher/standard/DefaultServiceEnricherTest.java

```diff
@@ -97,6 +97,17 @@ public void portConfigWithMultipleMappings() throws Exception {
         assertThat(json, hasJsonPath("$.spec.ports[*]", hasSize(2)));
     }
 
+
+    @Test
+    public void portConfigWithMultipleMapping1() throws JsonProcessingException {
+        setupExpectations("port", "8080:8081,8443:8443", "multiPort", "true", "normalizePort", "true");
+
+        String json = enrich();
+        assertPort(json, 0, 80, 8081, "http", "TCP");
+        assertPort(json, 1, 443, 8443, "https", "TCP");
+        assertThat(json, hasJsonPath("$.spec.ports[*]", hasSize(2)));
+    }
+
```

Comment:
Can you add a use case when normalize is `false`?

Reply:
The normalize is false by default... So all other test cases are normalize = false.